tests/test_diaForcedSource.py

```diff
@@ -202,7 +211,7 @@ def testRun(self):
 
         # Should be number of test objects minus one as one object is purposely
         # outside of the ccd area.
-        self.assertEqual(len(dia_forced_sources), len(self.testDiaObjects) - 3)
+        self.assertEqual(len(dia_forced_sources), len(self.testDiaObjects) - 2)
```

Comment:
I'm confused about this line and the comment. It looks like you're adding 3 sources outside the ccd in `create_test_dia_objects`, yet the comment says one and the comparison here looks like two.
After playing with the code a little, it looks like the selection of five sources on line 193 is including two of the three outside sources, and omitting the third. If I expand that to `[1:7]` toinclude 6 sources and also fix the source id of the third outside source, then this line needs to be `len(self.testDiaObjects) - 1` to pass, which matches the comment. I'm pretty sure I'm missing something here, though. Could you please double-check the logic, and either expand the comment to explain why there is a `-2` here, or modify the code if it is incorrect in some way?

Reply:
Made a variable in setup and added more comments for clarity (hopefully).